applications/nrf5340_audio/src/modules/led.c

```diff
@@ -269,32 +231,39 @@ int led_blink(uint8_t led_unit, ...)
 		return -EINVAL;
 	}
 
-	return led_set(led_unit, color, LED_BLINK);
-}
-
-int led_off(uint8_t led_unit)
-{
-	return led_set(led_unit, LED_COLOR_OFF, LED_SOLID);
+	return led_set(led, color, LED_BLINK);
```

Comment:
What happens when you blink a RGB LED?

Reply:
If it is just the red in an RGB LED then just the red will blink.
If it is a monochrome LED it blinks it regardless of colour.